R/FDistribution.R

```diff
@@ -0,0 +1,173 @@
+#' F-Distribution
```

Comment:
The Rd file is missing. Please run devtools::document() to create it.

Reply:
Added @include statements to ensure correct collate order of roxygen2, otherwise devtools::document() did not run. Should work now!